### DISCUSSION TITLE
Is this normal

### DISCUSSION THREAD

**Discussion by seckeymaker** [Security-Onion-Solutions/securityonion#12523] [General]
After install, is this normal ?

![IMG_20240307_132845](https://github.com/Security-Onion-Solutions/securityonion/assets/8999097/0ead3497-8bb6-4daa-b3dd-e880e9f49b94)

thank you

**Comment by cm-ops** [comment]
https://docs.securityonion.net/en/2.4/console.html#console those are messages from the Linux kernel.
